src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -126,13 +126,13 @@ def save_as_table(
             mode: One of the following values. When it's ``None`` or not provided,
                 the save mode set by :meth:`mode` is used.
 
-                "append": Append data of this DataFrame to existing data.
+                "append": Append data of this DataFrame to the existing table.
```

Comment:
Unrelated to your change: I think we should also mention that we will create the table if it does not exist.